pulpcore/cli/core/generic.py

```diff
@@ -71,6 +94,52 @@ def _task_group_filter_callback(
         help=_("List only tasks in this task group. Provide pulp_href or UUID."),
         callback=_task_group_filter_callback,
     ),
+    click.option(
+        "--parent-task",
+        help=_("Parent task by uuid or href."),
+        callback=_task_filter_callback,
+    ),
+    resource_option(
+        "--worker",
```

Comment:
We support worker__in as well - this should support multiple=True, yeah?

Reply:
I can try. Don't see much value though.

Reply:
The `worker__in` filter complains about "not a valid UUID" when it was supposed to accept hrefs.
I'm not going to implement that here.